pkg/scheduler/api/job_info.go

```diff
@@ -265,11 +269,11 @@ func (ti *TaskInfo) Clone() *TaskInfo {
 	}
 }
 
-func (ti *TaskInfo) GetTaskSpecKey() TaskID {
```

Comment:
Sorry, I don't understand why the TaskID type was changed to string?

Reply:
> Sorry, I don't understand why the TaskID type was changed to string?

It is just a string of spec role, has nothing to do with taskID. And the origin defination of spec is also a string.

Reply:
got it! thanks!